### PR TITLE
Consolidated Red and Blue templates

### PR DESCRIPTION
**Pull request by mshkolnik22**
## Description

(insert summary)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: templates/PURPLE.html [hunk 1]
```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>{{ page_title }} | CALDERA</title>
```
[the remark this answers is not in the document]
Comment (mshkolnik22) [reply templates/PURPLE.html:4]: Yes! it does look better!

**Review by argaudreau — DISMISSED** [review]
Works as expected, great work!

File: templates/core.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>Red | CALDERA</title>
+        <title>{{ page_title }} team | CALDERA</title>
```
Comment (iguannalin) [inline templates/core.html:4]: This shows up as i.e. 'Red Team team'

**Review by iguannalin — CHANGES_REQUESTED** [review]
One more fix and it looks good!

File: templates/core.html (hunk 2 above)
Comment (mshkolnik22) [reply templates/core.html:4]: Ah, great catch!

Comment (mshkolnik22) [reply templates/core.html]: I just made an update, could you please take another look?

**Pull request merged**